docs/getting-started.md

```diff
@@ -1,52 +0,0 @@
-## Getting Started
```

Comment:
out of interest, why did we move from using this copy to the one here https://github.com/Kuadrant/docs.kuadrant.io/blob/main/docs/getting-started.md?